packages/main/test/specs/ComboBox.spec.js

```diff
@@ -283,6 +283,30 @@ describe("General interaction", () => {
 
 	});
 
+	it("should fire change event after the user has typed in value, but also selects it from the popover", async () => {
+        await browser.url(`test/pages/ComboBox.html`);
+
+		// Setup
+		const changeValue = await browser.$("#change-placeholder");
+        const counter = await browser.$("#change-count");
+        const combo = await browser.$("#change-cb");
+		const input = await combo.shadow$("[inner-input]");
+
+
+		// Type something which is in the list
+		await input.click();
+		await input.keys("Bulgaria");
+
+		// Click on the item
+		const staticAreaItemClassName = await browser.getStaticAreaItemClassName("#change-cb");
+		const popover = await browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover");
+		await (await popover.$("ui5-list").$$("ui5-li")[0]).click();
+
```

Comment:
remove the line.

Reply:
Why? How would that improve the code or the behaviour?

Reply:
It improves the look of the code, since it's a rule of thumb to have one empty line between the different code blocks.
Also, I don't understand why once you use one empty line and after that you use two.